src/domUpdates.js

```diff
@@ -19,9 +19,8 @@ const availableRoomsSection = document.getElementById('availableRooms');
 const filterButton = document.querySelector('.filter-button');
 const typeFilters = document.querySelectorAll('.checkbox-js');
 
-const qs = {
-  dashboardMenuButton,
-  bookMenuButton
+const querySelectors = {
```

Comment:
Definitely better naming convention for someone who might be reading your code 👩‍💻